AnnService/src/IndexSearcher/main.cpp

```diff
@@ -63,6 +65,10 @@ class SearcherOptions : public Helper::ReaderOptions
     int m_genTruth = 0;
 
     int m_debugQuery = -1;
+
+    std::string m_quantizerFile;
```

Comment:
QuantizerFile as a setting can be removed, it is from before I add the load with index. I think ADC should be option here though, since it is a search property not an index property